cmd/tikv-ctl/src/executor.rs

```diff
@@ -56,6 +57,27 @@ pub const LOCK_FILE_ERROR: &str = "IO error: While lock file";
 
 type MvccInfoStream = Pin<Box<dyn Stream<Item = result::Result<(Vec<u8>, MvccInfo), String>>>>;
 
+fn get_engine_type(dir: &str) -> EngineType {
+    let mut entries = read_dir(dir).unwrap();
+    let mut engine1 = false;
+    let mut engine2 = false;
+    while let Some(Ok(e)) = entries.next() {
+        if let Ok(ty) = e.file_type() && ty.is_dir() {
+            if e.file_name() == "tablets" {
+                engine2 = true;
+            } else if e.file_name() == "db" {
+                engine1 = true;
+            }
+        }
+    }
+    assert_ne!(engine1, engine2);
+    if engine1 {
```

Comment:
Does it need to return `raftKv` if there is no `tables` or `db`?

Reply:
I think it only occurs when cluster has never been started, and in this case, it will be panic at line 73.